WordPress/Classes/ViewRelated/Comments/Content/CommentContentHeaderView.swift

```diff
@@ -4,6 +4,13 @@ import DesignSystem
 struct CommentContentHeaderView: View {
 
     let config: Configuration
+    var menu: MenuConfiguration {
+        return config.menu
+    }
+
+    private var shouldShowMenu: Bool {
+        return [menu.userInfo, menu.share, menu.editComment, menu.changeStatus].reduce(false) { $0 || $1 }
```

Comment:
Just curious if this function checks whether one of the menu items is `true`. 
`.reduce(false) { $0 || $1 }` part seems to be not entirely intuitive, and I'm wondering if `.contains { _ in true }` would have a similar result.

Reply:
Resolved in 7154ae579a96a920ecd563c928d777660817c75e